src/control/server/storage/scm/ipmctl_test.go

```diff
@@ -116,9 +116,48 @@ func newMockIpmctl(cfg *mockIpmctlCfg) *mockIpmctl {
 	}
 }
 
-// TestGetState tests the internals of ipmCtlRunner, pass in mock runCmd to verify
+// TestIpmctl_checkIpmctl verified that bad versions trigger an error.
+func TestIpmctl_checkIpmctl(t *testing.T) {
```

Comment:
I'm concerned about this test being brittle. I don't think we should be testing specific real versions here, and instead the code should be refactored as suggested previously in order to compare a version string against an arbitrary list of version strings.

As it's implemented currently, we will have to keep adding new versions to ensure that the matching code works correctly, when instead we could just synthesize a variety of versions in order to be satisfied that the code works correctly.